src/middlewared/middlewared/restful.py

```diff
@@ -497,9 +518,13 @@ def get_post_args(self, req, resp, **kwargs):
                     filterid = kwargs['id']
                     if filterid.isdigit():
                         filterid = int(filterid)
+                    extra = {}
+                    for key, val in list(req.query.items()):
```

Comment:
Should it be `extra.` here too?

Reply:
I don't think so because we are not expecting anything else here ?